README.md

```diff
@@ -59,34 +60,34 @@ Realistic example, also uses a sibling plugin [shipkit-auto-version](https://git
 
     tasks.named("generateChangelog") {
         previousRevision = project.ext.'shipkit-auto-version.previous-tag'
-        readOnlyToken = "1234c0f41c200f7c653323014d6a72a127764e17"
+        githubToken = System.getenv("GITHUB_TOKEN") // using env var to avoid checked-in secrets
         repository = "shipkit/shipkit-changelog"
     }
     
     tasks.named("githubRelease") {
         dependsOn tasks.named("generateChangelog")
         repository = "shipkit/shipkit-changelog"
-        changelog = tasks.named("generateChangelog").get().outputFile                
-        writeToken = System.getenv("GH_WRITE_TOKEN")
+        changelog = tasks.named("generateChangelog").get().outputFile
+        githubToken = System.getenv("GITHUB_TOKEN") // using env var to avoid checked-in secrets
+        newTagRevision = System.getenv("GIT_SHA")   // using an env var automatically exported by GH action
     }
 ```
 
 ## Configuration reference
 
 ### GitHub access tokens
 
-The standard way to enable automated tasks read/write to GitHub are [personal access tokens](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token#creating-a-token).
-When creating the tokens, please select the following token **scopes** ([more info on scopes](https://docs.github.com/en/free-pro-team@latest/developers/apps/scopes-for-oauth-apps)):
+The standard way to enable the automated tasks read/write to GitHub are the
+[personal access tokens](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token#creating-a-token).
+Shipkit Changelog plugin needs a token to fetch tickets and post releases via GH REST API. 
+The token is set in the task configuration in \*.gradle file via `githubToken` property.
+When creating a new token using GH UI, make sure to select the `repo/public_repo` *scope*
+([more info on scopes](https://docs.github.com/en/free-pro-team@latest/developers/apps/scopes-for-oauth-apps)).
 
-When using GH actions you can use [GITHUB_TOKEN](https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow) 
-for `readOnlyToken` and `writeToken`
-
-- readOnlyToken - should have **no scope**, this way it only provides read-only access to **public** repositories
-(it **does not** provide read-only access to private repositories).
-  
-  You can leave this field empty for local development, in this case you are limited to 60 requests per hour.  
-
-- writeToken - needs 'repo/public_repo' scope to post releases via GH REST API.
+When using GH Actions then `GITHUB_TOKEN` environment variable is exported *by default*.
```

Comment:
`GITHUB_TOKEN` isn't exported by default, we must use it step. It is avaliable in github.token context and as secret secrets.GITHUB_TOKEN, but not as environment variable.